files/scripts/swss.sh

```diff
@@ -83,6 +84,9 @@ startPeerService() {
 
     if [[ x"$WARM_BOOT" != x"true" ]]; then
         /bin/systemctl start ${PEER}
+        for dep in ${DEPENDENT}; do
```

Comment:
suggest to change the function name to startPeerService to startPeerDependentServices

Reply:
Thanks. Done!